exercises/00-init/src/modules/users/users-saga.js

```diff
@@ -0,0 +1,35 @@
+import { call, fork, put, takeEvery } from 'redux-saga/effects';
+
+import {getUsers as getUsersEffect, addUser as addUserEffect} from 'modules/users/users-effects'
+import UsersActions from 'modules/users/users-actions'
+
+
+function* getUsers() {
+    try {
+        const users = yield call(getUsersEffect);
+        yield put({ type: UsersActions.Types.LOADED, payload: users });
+    }
+    catch (e)
+    {
+        console.log(`getUsers Saga failed:${e}`)
+    }
+}
+
+function* addUser(user) {
+    try {
+        yield call(addUserEffect, user.firstName, user.lastName)
+        yield getUsers();

```

Comment:
No reason not to use `call` here as well :)

Reply:
Well, I didn't particularly care about waiting in this case. In that case I could use fork, to make it explicit. Both work in this case, with no relevant difference, so I'll stick with call.